pkg/engine/vmextensions.go

```diff
@@ -86,6 +86,7 @@ func CreateCustomScriptExtension(cs *api.ContainerService) VirtualMachineExtensi
 			AutoUpgradeMinorVersion: to.BoolPtr(true),
 			Settings:                &map[string]interface{}{},
 			ProtectedSettings: &map[string]interface{}{
+				//note that any change to this property will trigger a CSE rerun on upgrade as well as reimage. 
```

Comment:
This inline comment in a struct literal expression is breaking lint. gofmt should put you on the right path